src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -56,9 +56,18 @@ public class SeedDb extends DatastoreClient {
     private static final int MAX_QUESTION_PER_COURSE = 6;
     private static final int MAX_RESPONSES_PER_QUESTION = 10;
     private static final int MAX_COMMENTS_PER_RESPONSE = 2;
+    private static final int NOTIFICATION_SIZE = 1000;
+    private static final int READ_NOTIFICATION_SIZE = 5;
```

Comment:
Do notifs and read notifs need to be seeded for our tests?

Reply:
Not really but I left it since its part of the seeddb script, it doesn't take too much time to seed so I think its ok